mod-kb-ebsco/mod-kb-ebsco-java.postman_collection.json

```diff
@@ -7732,7 +7821,7 @@
 											"raw": ""
 										},
 										"url": {
-											"raw": "{{protocol}}://{{url}}:{{okapiport}}/eholdings/packages/{{packageId}}?include=",
+											"raw": "{{protocol}}://{{url}}:{{okapiport}}/eholdings/packages/{{packageId}}?include=prov",
```

Comment:
please remove `prov` value, seems it vas a small typo.

Reply:
Thanks, @natali-zaitseva. Done.